applications/tari_console_wallet/src/ui/components/menu.rs

```diff
@@ -0,0 +1,65 @@
+use crate::ui::{components::Component, state::AppState};
+use tui::{
+    backend::Backend,
+    layout::{Constraint, Direction, Layout, Rect},
+    style::{Color, Modifier, Style},
+    text::{Span, Spans},
+    widgets::{Block, Paragraph},
+    Frame,
+};
+
+pub struct Menu {}
+
+impl Menu {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl<B: Backend> Component<B> for Menu {
+    fn draw(&mut self, f: &mut Frame<B>, area: Rect, _app_state: &AppState)
+    where B: Backend {
+        let columns = Layout::default()
+            .direction(Direction::Horizontal)
+            .constraints([Constraint::Ratio(4, 5), Constraint::Ratio(1, 5)].as_ref())
+            .split(area);
+
+        let others = Spans::from(vec![
+            Span::styled("LeftArrow", Style::default().fg(Color::Green)),
+            Span::styled(":", Style::default().fg(Color::White)),
+            Span::styled(
+                "PrevTab ",
+                Style::default()
+                    .fg(Color::Magenta)
+                    .bg(Color::LightGreen)
+                    .add_modifier(Modifier::BOLD),
+            ),
+            Span::raw(" "),
+            Span::styled("Tab/RightArrow", Style::default().fg(Color::Green)),
```

Comment:
```suggestion
            Span::styled("Tab/RightArrow ", Style::default().fg(Color::Green)),
```

Reply:
The space at the end is a styling thing on the TUI. Maybe we should get a vote on it; either way can work for me.

Reply:
Sorry I actually meant a space after the colon for both

![image](https://user-images.githubusercontent.com/351403/112474128-e5be2300-8d77-11eb-94a6-ee71ece16bf0.png)